agora-test/Spec/Sample/Governor.hs

```diff
@@ -0,0 +1,531 @@
+{- |
+Module     : Spec.Sample.Governor
+Maintainer : connor@mlabs.city
+Description: Sample based testing for Governor utxos
+
+This module tests primarily the happy path for Governor interactions
+-}
+module Spec.Sample.Governor (
+  createProposal,
+  mutateState,
+  mintGATs,
+  mintGST,
+) where
+
+--------------------------------------------------------------------------------
+
+import Plutarch.Api.V1 (mkValidator, validatorHash)
+import Plutarch.SafeMoney.Tagged
+
+--------------------------------------------------------------------------------
+
+import Plutus.V1.Ledger.Address (scriptHashAddress)
+import Plutus.V1.Ledger.Api (
+  Address (..),
+  Credential (PubKeyCredential),
+  Datum (..),
+  PubKeyHash,
+  ScriptContext (..),
+  ScriptPurpose (Minting, Spending),
+  ToData (toBuiltinData),
+  TokenName (..),
+  TxInInfo (TxInInfo),
+  TxInfo (..),
+  TxOut (..),
+  TxOutRef (..),
+  Validator,
+  ValidatorHash (..),
+ )
+import Plutus.V1.Ledger.Interval qualified as Interval
+import Plutus.V1.Ledger.Scripts (unitDatum)
+import Plutus.V1.Ledger.Value (
+  AssetClass (..),
+ )
+import Plutus.V1.Ledger.Value qualified as Value
+import PlutusTx.AssocMap qualified as AssocMap
+
+--------------------------------------------------------------------------------
+
+import Agora.Effect.NoOp
+import Agora.Governor
```

Comment:
Unless qualified _with a unique name_, all imports should be strict i.e. you need to specific exactly _what_ you're importing from a module.